documentation/deployment.md

```diff
@@ -4,6 +4,14 @@
 
 The only dependecy for perseo-core is the servlet engine container for its WAR file, in particular, a Tomcat 7.
 
+### Installation using Docker
+
+Every development version of Perseo Core is uploaded to Docker Hub for every PR merged into the `develop` branch.
+In order to start Perseo Core as a deamonized Docker component use the following command:
+```
+docker run -d fiwareiotplatform/perseocore
+```
+
```

Comment:
Should be great to have the same (great) build info that is in the Orion readme like: 

## Build the image

You only need to do this once in your system:

	docker build -t perseo .

The parameter `-t perseo` gives the image a name. This name could be anything, or even include an organization like `-t org/fiware-perseo`. This name is later used to run the container based on the image.

If you want to know more about images and the building process you can find it in [Docker's documentation](https://docs.docker.com/userguide/dockerimages/).
    
## Run the container

The following line will run the container exposing port `8080`, give it a name -in this case `perseo1`, link it to a mongodb docker, and present a bash prompt.

	  docker run -d --name perseo1 --link mongodb:mongodb -p 8080:8080 perseo -dbhost mongodb

As a result of this command, there is a PERSEO listening on port 8080 on localhost. Try to see if it works now with

	curl localhost:8080/version

A few points to consider:

* The name `perseo1` can be anything and doesn't have to be related to the name given to the docker image in the previous section.
* `--link mongodb:mongodb` assumes there is a docker container running a MongoDB image in your system, whose name is `mongodb`. In case you need one type `docker run --name mongodb -d mongo:2.6`.
* In `-p 8080:8080` the first value represents the port to listen in on localhost. If you wanted to run a second context broker on your machine you should change this value to something else, for example `-p 9090:8080`.
* Anything after the name of the container image (in this case `perseo`) is interpreted as a parameter for the Perseo CEP. In this case we are telling the broker where the MongoDB host is, represented by the name of our other MongoDB container. Take a look at the [documentation](../.../.../doc/admin/cli.md) for other command-line options.